universal-login-react/src/core/services/DeleteAccountService.ts

```diff
@@ -20,11 +20,23 @@ export const getInputClassName = (inputError: boolean) => inputError ? 'delete-a
 
 const doesAnyErrorExists = (errors: ErrorsType) => errors.usernameError || errors.verifyFieldError;
 
-export const deleteAccount = async (walletService: WalletService, inputs: InputsType, setErrors: (errors: ErrorsType) => void, onAccountDeleted: () => void) => {
+export const deleteAccount = async (
+  walletService: WalletService,
+  inputs: InputsType,
+  setErrors: (errors: ErrorsType) => void,
+  onAccountDeletion: (transactionHash?: string) => void,
+  onAccountDeleted: () => void,
+) => {
   const errors = checkInputsAgainstError(walletService.getDeployedWallet().name, inputs);
   setErrors(errors);
   if (!doesAnyErrorExists(errors)) {
-    await walletService.removeWallet(transactionDetails);
+    onAccountDeletion();
```

Comment:
Maybe change the name to `onWaitingForDeleteAccount`

Reply:
Also, agree.